tests/client.rs

```diff
@@ -134,7 +131,7 @@ async fn test_directory_not_existing() {
     let root_cause = error.root_cause();
     assert_eq!(
         format!("{}", root_cause),
-        "Corrupt cache: directory cache does not exist"
+        "No such file or directory (os error 2)"
```

Comment:
this looks like a system error to me, but this test was meant to test the case that a missing directory cache is handled explicitely by the program (i.e. gives an exact error message to the user)

Reply:
Ah, OK, now I understand better what you wrote in the chat :) I'll adapt the test to return the more informative message.